src/libzerocoin/bignum.h

```diff
@@ -10,6 +10,7 @@
 #include "veil-config.h"
 #endif
 
+#include <limits.h>
```

Comment:
This looks to be successful (limits must have been getting pulled in somewhere down the path of compat.h).

Let's put this into a separate PR so we can fast-track it through before anything else.